recipes/cffi/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "cffi" %}
+{% set version = "1.7.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "6ed5dd6afd8361f34819c68aaebf9e8fc12b5a5893f91f50c9e50c8886bb60df" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+  patches:
+    - setup-linux.patch  # [unix]
```

Comment:
We should add a PR or issue with this patch upstream. Though it may just work with `pkg-config` or at least that is what it tries.